src/signals/IncidentMap/components/AddressLocation/AdressLocation.test.tsx

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+
+import { render, screen } from '@testing-library/react'
+
+import { formatAddress } from '../../../../shared/services/format-address'
+import type { Props } from './AddressLocation'
+import { AddressLocation } from './AddressLocation'
+
+const coords = {
+  lat: 52.3731081,
+  lng: 4.8932945,
+}
+
+const mockPdokAddress = {
+  huisnummer: '178',
+  openbare_ruimte: 'Warmoesstraat',
+  postcode: '1012JK',
+  woonplaats: 'Amsterdam',
+}
+
+const mockPdokResponse = {
+  id: 1,
+  value: 'mockValue',
+  data: {
+    location: coords,
+    address: mockPdokAddress,
+  },
+}
+
+jest.mock(
+  'signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/styled',
+  () => ({
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    ...jest.requireActual(
+      'signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/styled'
+    )!,
```

Comment:
If you remove the "!" the comment is not needed.

Reply:
Hoe kom je daar achter?